app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -1,35 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".ui.signin.SignInActivity">
+    android:layout_width="match_parent" android:layout_height="match_parent"
+    android:background="@color/colorPrimary">
 
-    <EditText
-        android:id="@+id/editText"
+
+    <ImageView
+        android:id="@+id/sigin_image_logo"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:ems="10"
-        android:hint="@string/hint_email"
-        android:inputType="textEmailAddress"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
-
-    <EditText
-        android:id="@+id/editText2"
+        android:layout_marginTop="80dp"
+        android:layout_centerHorizontal="true"
+        android:src="@mipmap/signin_logo" />
+
+    <TextView
+        android:id="@+id/signin_text_almost_there"
+        android:layout_marginTop="16dp"
```

Comment:
please remove hardcoded string and do take it from dimes.xml file

Reply:
These changes are from #10 from @derrickabreo, he made the commits before we updated our coding style guidelines. This is not a shows stopper or breaking change. We can take it up later, right now the priority is  making the development branch stable.  @sagarsrao please create an issue and note these points there.

Reply:
Will do that.Thanks